### ISSUE TITLE
18.04 Installer Crashes

### ISSUE THREAD

**Issue by Rottenstagg** [Distroshare/distroshare-ubuntu-imager#19]
Installer crashes when trying to open it and freeze all system.

Error: **Ubiquity : unable to determine the release**

Version: Xubuntu 18.04

**Comment by bobbydoogle** [comment]
I had possibly the same issue (customizing 18.04 ubuntu vanilla), made some minimal changes for testing, installed VScode / tweaks and changed a wallpaper navbar location. Looks like the scripts all ran correctly, live cd was generated, then I ran it in virtualbox, live desktop looked fine (except my customizations were gone, vscode was there), going through installer from live desktop it would crash when entering user info / credentials every time. Is this project supported, I see it hasn't been updated in some time.

**Comment by danimesq** [comment]
@hugegreenbug

**Comment by RealistikDash** [comment]
Yes, having the same issue here. Additionally my gnome tweaks extensions didn't carry over.

**Comment by cinnabarLinux** [comment]
Had this issue as well. Very frustrating. I LOVE this tool and everything it stands for, but obviously this is a show stopping bug that makes the whole tool virtually useless... aside from creating Live images that cannot be installed. Really hope this gets fixed at some point. Might even fix it myself if I have time. I already tried using Anaconda instead of Ubiquity and got the same crash at user creation menu :(

**Comment by samuelkurt** [comment]
I think I found a Solution
1) Create the Image file using Distroshare
2) Then Install custom Ubuntu ISO Creator with this commands:
sudo apt-add-repository ppa:cubic-wizard/release
sudo apt update
sudo apt install --no-install-recommends cubic
3) Open custom Ubuntu ISO Creator and select any Folder (Output)
4) Choose your Distroshare ISO
5) Click Next till the End, leave everything as it is
6) Check delete all files except Output (optional for cleaner output)
7) Go to the Outputfolder and copy that ISO anywhere
8) Enjoy!
